<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog{
	background-image: url(:/MouseDrv/skins/msg_nr.png);
}
QLabel{
    color: lightgray;
}
QPushButton{
    color: lightgray;
    font: bold 10pt;
    border:none;
    background:url(:/MouseDrv/skins/btn_nr.png);
}
QPushButton:hover{
    background:url(:/MouseDrv/skins/btn_ov.png);
}
#btnExit{
    background:url(:/MouseDrv/skins/exit_nr.png);
}
#btnExit:hover{
    background:url(:/MouseDrv/skins/exit_ov.png);
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>351</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>197</y>
     <width>88</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>197</y>
     <width>88</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTips">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>23</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tips</string>
   </property>
  </widget>
  <widget class="QWidget" name="centerWidget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>59</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="mousedrv.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
